src/OrchardCore/OrchardCore.OpenId.Core/YesSql/Migrations/OpenIdMigrations.cs

```diff
@@ -62,21 +70,24 @@ public int UpdateFrom1()
 
         public int UpdateFrom2()
         {
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByPostLogoutRedirectUriIndex");
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRedirectUriIndex");
-            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRoleNameIndex");
+            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByPostLogoutRedirectUriIndex", null);
+            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRedirectUriIndex", null);
+            SchemaBuilder.DropReduceIndexTable("OpenIdApplicationByRoleNameIndex", null);
 
-            SchemaBuilder.CreateReduceIndexTable(nameof(OpenIdAppByLogoutUriIndex), table => table
+            SchemaBuilder.CreateReduceIndexTable(typeof(OpenIdAppByLogoutUriIndex), table => table
                 .Column<string>(nameof(OpenIdAppByLogoutUriIndex.LogoutRedirectUri))
-                .Column<int>(nameof(OpenIdAppByLogoutUriIndex.Count)));
+                .Column<int>(nameof(OpenIdAppByLogoutUriIndex.Count)),
+                null);
 
-            SchemaBuilder.CreateReduceIndexTable(nameof(OpenIdAppByRedirectUriIndex), table => table
+            SchemaBuilder.CreateReduceIndexTable(typeof(OpenIdAppByRedirectUriIndex), table => table
                 .Column<string>(nameof(OpenIdAppByRedirectUriIndex.RedirectUri))
-                .Column<int>(nameof(OpenIdAppByRedirectUriIndex.Count)));
+                .Column<int>(nameof(OpenIdAppByRedirectUriIndex.Count)),
+                null);
```

Comment:
the collection name parameter is optional, so I would not pass null. The intent was for it to be non breaking, so we shouldn't need to change this. Just the types.

Can you use the generics everywhere?

Reply:
ok